schematics/collection.json

```diff
@@ -1,5 +1,12 @@
 // This is the root config file where the schematics are defined.
 {
   "$schema": "./node_modules/@angular-devkit/schematics/collection-schema.json",
-  "schematics": {}
+  "schematics": {
+    "materialShell": {
```

Comment:
FYI we're probably going to change this name at some point but it's still TBD

Reply:
This should be `ng-add`.

Reply:
Adding an alias for it gives this error: `Error: Schematics/alias "ng-add" collides with another alias or schematic name.`